python/pdstools/infinity/resources/prediction_studio/v24_2/prediction.py

```diff
@@ -0,0 +1,386 @@
+from __future__ import annotations
+
+from datetime import date
+from typing import Dict, List, Literal, Optional, Union, overload
+
+import polars as pl
+from pdstools import cdh_utils
+from pydantic import validate_call
+
+from ....internal._constants import METRIC
+from ....internal._exceptions import NoMonitoringInfo, PegaException, PegaMLopsError
+from ....internal._pagination import PaginatedList
+from ..base import Notification
+from ..v24_1.prediction import Prediction as PredictionPrevious
+from .champion_challenger import ChampionChallenger
+from .model import Model
+from ..types import NotificationCategory
+
+
+class Prediction(PredictionPrevious):
+    """
+    The `Prediction` class provide functionality including retrieving notifications, models,
+    adding conditional models, getting champion challengers, metrics, and plotting metrics.
+    """
+
+    @overload
+    def get_notifications(
```

Comment:
Should we call this `list_notifications` just like we have `list_models` and `list_predictions`?